### DISCUSSION TITLE
Add a setting in Rome to set the maximum severity level

### DISCUSSION THREAD

**Discussion by flexiworld** [rome/tools#4584] [Suggestions]
I would like to be able to distinguish at a glance between the tsc compiler errors and the Rome linter ones in vscode.
So it would be great to have a setting in the Rome linter so that the maximum severity level reported to vscode is "warn" instead of "error".

**Comment by ematipico** [comment]
You can change the level of a rule already: https://docs.rome.tools/linter/#change-the-diagnostic-severity

If that's not what you're suggesting, please share more details

**Reply by flexiworld** [reply]
I don't want to set it individually for each and every one of the 141 rules. That is just not a reasonable effort to write and maintain.
What I would like is one global setting that apply to all the linter rules.
Also, it's about applying a ceiling severity level. I want rules that are "info" severity level to still stay at "info".

**Comment by flexiworld** [comment]
There is already a plugin for eslint that does exactly what I am suggesting here.
It's called eslint-plugin-only-warn: https://github.com/bfanger/eslint-plugin-only-warn
